src/test/java/calculator/service/impl/CalculatorServiceImplTest.java

```diff
@@ -0,0 +1,46 @@
+package calculator.service.impl;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.math.BigInteger;
+import org.junit.jupiter.api.Test;
+
+class CalculatorServiceImplTest {
+
+    private CalculatorServiceImpl calculatorService = new CalculatorServiceImpl();;
+
+    @Test
+    void simpleCalculateTest() {
+        BigInteger[] numbers = {BigInteger.ONE, BigInteger.valueOf(2), BigInteger.valueOf(3), BigInteger.valueOf(4), BigInteger.valueOf(5)};
+        BigInteger result = calculatorService.calculate(numbers);
+        assertEquals(BigInteger.valueOf(15), result);
```

Comment:
AssertJ 방식의 테스트 코드 작성은 어떻게 생각하시나요?!

```suggestion
        assertThat(result).isEqualTo(BigInteger.valueOf(15));
```

저는 가독성 측면에서 이점이 있다고 생각해 해당 방법을 적극적으로 사용하고 있어요